vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -94,6 +93,12 @@ nginx.ingress.kubernetes.io/proxy-buffer-size: 64k # for the Keycloak auth-token
 nginx.ingress.kubernetes.io/proxy-read-timeout: "600" # to increase the timeout when uploading large files
 ```
 
+Sticky sessions may be useful for `entando-de-app` deployments with multiple replicas. If you have [set up clustering](caching-and-clustering.md/#clustering), the following options will enable sticky sessions in NGINX:
```

Comment:
"If you have [set up clustering](caching-and-clustering.md/#clustering), the following options..." a little awkward at the beginning>> maybe 'if you are setting up the cluster' or 'When you set up the clustering'. It seems odd to say you 'have' set up clustering.

Reply:
hm the key point was for the link verbiage to be "set up clustering," and to convey that clustering was set up prior. "when" can imply that it's a given or expected that you use clustering, when really it's just an option. i think we can just remove "have"... "If you set up clustering" works just as well for past tense.

Reply:
I think it is only awkward because of the link highlight, so 'set up clustering' sounds like a thing. When I read it as part of the verb phrase 'have set up' as you meant, it's perfectly fine. But now it reads better and is shorter:)